messages/npm.lerna.release.json

```diff
@@ -6,6 +6,7 @@
   "npmAccess": "access level to use when publishing to npm",
   "install": "run yarn install and build on repository",
   "githubRelease": "create release in github based on the package changes",
+  "verify": "verify npm registry has new version after publish and digital signature",
```

Comment:
I think this would be more accurate:

```
verify that the npm registry has the new version after publish. If the package is signed, also verify that the digital signature exists
```

Or something like that

Reply:
@mshanemc Verification of digital signature is only done if the plugin has been signed.